docs/user-guide/cli-using-team-configuration-application-developers.md

```diff
@@ -1,30 +1,40 @@
-# Team configuration for application developers
+# Configuration for application developers
```

Comment:
@adam-wolfe @zFernand0 @sbohnert - please do not review this page b/c it's not going into the V3 site's final version. This content was broken up and re-organized into other pages included in this PR.